cmd/wallet-web/src/pages/Signin.vue

```diff
@@ -12,7 +12,7 @@
       px-6
       md:bg-onboarding
       flex flex-col
-      min-w-screen min-h-screen
```

Comment:
Can you, please, post a screenshot of this component with the change?

Reply:
This was because of the warning saying no such class as min-w-screen in tailwind so i removed, and checked no change in the screen.